dist/monet.min.js

```diff
@@ -6,5 +6,5 @@
ue;throw"Illegal state. Cannot call right() on a Either.left"},left:function(){if(this.isRightValue)throw"Illegal state. Cannot call left() on a Either.right";return this.value},cata:function(t,n){return this.isRightValue?n(this.value):t(this.value)},equals:function(t){return isFunction(t.isRight)&&isFunction(t.cata)?this.cata(function(n){return t.cata(equals(n),falseFunction)},function(n){return t.cata(falseFunction,equals(n))}):!1},bimap:function(t,n){return this.isRightValue?this.map(n):this.leftMap(t)},toMaybe:function(){return this.isRight()?L(this.value):v()},toValidation:function(){return this.isRight()?N(this.value):F(this.value)},toString:function(){return this.cata(function(t){return"Left("+t+")"},function(t){return"Right("+t+")"})},inspect:function(){return""+this}},q.fn.init.prototype=q.fn;var A,O=A=n.Reader=function(t){return new O.fn.init(t)};O.of=function(t){return O(function(n){return t})},O.ask=function(){return O(idFunction)},O.fn=O.prototype={init:function(t){this.f=t},run:function(t){return this.f(t)},bind:function(t){var n=this;return O(function(i){return t(n.run(i)).run(i)})},ap:function(t){var n=this;return t.bind(function(t){return O(function(i){return t(n.run(i))})})},map:function(t){var n=this;return O(function(i){return t(n.run(i))})},local:function(t){var n=this;return O(function(i){return n.run(t(i))})}},O.fn.init.prototype=O.fn;var j=n.Free={},z=j.Suspend=n.Suspend=function(t){return new j.fn.init(t,!0)},C=j.Return=n.Return=function(t){return new j.fn.init(t,!1)};return j.of=function(t){return C(t)},j.liftF=function(t){return z(isFunction(t)?compose(C,t):t.map(C))},j.fn=j.prototype={init:function(t,n){this.isSuspend=n,n?this.functor=t:this.val=t},run:function(){return this.go(function(t){return t()})},bind:function(t){return this.isSuspend?z(isFunction(this.functor)?compose(function(n){return n.bind(t)},this.functor):this.functor.map(function(n){return n.bind(t)})):t(this.val)},ap:function(t){return this.bind(function(n){return t.map(function(t){return t(n)})})},resume:function(){return this.isSuspend?_(this.functor):I(this.val)},go1:function(t){function go2(n){return n.resume().cata(function(n){return go2(t(n))},idFunction)}return go2(this)},go:function(t){for(var n=this.resume();n.isLeft();){var i=t(n.left());n=i.resume()}return n.right()}},j.fn.init.prototype=j.fn,n.Identity=Identity,Identity.of=function(t){return new Identity(t)},Identity.fn=Identity.prototype={init:function(t){this.val=t},bind:function(t){return t(this.val)},get:function(){return this.val},equals:function(t){return isFunction(t.get)&&equals(this.get())(t.get())},toString:function(){return"Identity("+this.val+")"},inspect:function(){return""+this}},Identity.fn.init.prototype=Identity.fn,decorate(R),decorate(q),decorate(d),decorate(V),decorate(NEL),decorate(List),decorate(g),decorate(O),decorate(j),decorate(Identity),d.fromNull(t).filter(function(t){return!!t.notUseMonetGlobalObject}).cata(function(){return r(e,n)},function(t){return r(t,n),e})});
```

Comment:
I'll drop it in future (like just before release of `0.9.0` stable ) -- it just helped me to push things faster during development